packages/client/manifest.json

```diff
@@ -5186,5 +5186,88 @@
         "suffix": "repeater"
       }
     ]
+  },
+  "rowexplorer": {
+    "block": true,
+    "name": "Row explorer block",
```

Comment:
Could we use camel case here for the block name? All components use camel case, but there's inconsistency among the blocks. I think they should all be camel case. (Feel free to quickly update the other block titles to be camel case too if you want to, or we can do it separately).

Reply:
Do you literally just mean the name? like "Row Explorer block"? That's what I did, let me know if that's not what you meant 👍